groups/cdn-terraform/variables.tf

```diff
@@ -0,0 +1,48 @@
+variable "aws_account" {
+  type        = string
+  description = "The name of the AWS account"
+}
+
+variable "aws_region" {
+  type        = string
+  description = "The AWS region in which resources will be created"
+}
+
+variable "bucket_name" {
+  type        = string
+  description = "Name of the S3 bucket"
+}
```

Comment:
The bucket name is interpolated from several variables in the `aws_s3_bucket.s3_bucket` resource so this variable is redundant and can be removed:

```suggestion
```